deploy/server.conf

```diff
@@ -1,46 +1,48 @@
 server {
+
+    # Set the root to the public folder
+    # Node app routes are handled separately via passenger
     root /var/www/biglotteryfund/public;
+
+    # Enable Passenger
+    # APP_ENV is a placeholder and is replaced at deploy time
+    # with the appropriate environment value
     passenger_enabled on;
     passenger_startup_file bin/www;
     passenger_app_type node;
     passenger_app_env APP_ENV;
+
+    # Custom access log
+    # Access log is streamed to CloudWatch
+    # see cloudwatch-agent.json
     access_log /var/log/nginx/access.log;
 
-    # tweak headers
-    more_clear_headers      'Server' 'X-Powered-By' 'X-Runtime';
+    # Set common headers
     proxy_pass_header       Set-Cookie;
     add_header              X-Forwarded-Proto $http_x_forwarded_proto;
     add_header              X-Forwarded-For $proxy_add_x_forwarded_for;
 
-    # gzip everything
+    # Clear unwanted headers
+    # Enabled by Headers More include via the nginx-extras package
+    more_clear_headers      'Server' 'X-Powered-By' 'X-Runtime';
+
+    # Enable gzip
+    # This is largely redundant as CloudFront does automatic gzipping
+    # But we keep this enabled as a sensible default
     gzip_vary on;
     gzip_proxied any;
     gzip_comp_level 6;
     gzip_buffers 16 8k;
     gzip_http_version 1.1;
     gzip_types text/plain text/css application/json application/javascript text/xml;
 
+    # Set rate-limiting for user routes
+    # user_routes is defined in nginx.conf
     location ~ ^/user {
         limit_req zone=user_routes burst=20;
     }
 
     location ~ ^/welsh/user {
         limit_req zone=user_routes burst=20;
     }
-
-    # Media: images, icons, video, audio, HTC
```

Comment:
Given that everything under `/assets` is sent directly to S3 this block should never get executed.

Reply:
True, I guess this only gets called when accessing the site directly, but we can live with that not being cached etc (everything else isn't anyway) 👍

Reply:
Good point, happy to keep this in to account for that use-case?